app/pandas.ipynb

```diff
@@ -208,18 +413,92 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
```

Comment:
The 'Average age' is being calculated and added as a new column with the same value for each row. This is not typically how an average is represented in a DataFrame. Consider calculating the average separately and not adding it as a column.